databricks/koalas/series.py

```diff
@@ -2440,6 +2440,76 @@ def sort_index(
         else:
             return first_series(kdf)
 
+    def swaplevel(self, i=-2, j=-1, copy=True) -> "Series":
```

Comment:
Could you add a type annotation for `copy`?

Reply:
Sounds good, added!